src/factories/Category.php

```diff
@@ -26,5 +26,19 @@ function definition(int $index = 0)
             'groupId' => $group->id,
         ]);
     }
-
+    
+    /**
+     * @inheritdoc
+     */
+    protected function setAttributes($attributes, $element)
+    {
+        /** @var \craft\elements\Category $element */
```

Comment:
Can we move the type up to the method declaration?

```suggestion
     * @inheritdoc
     * @param \craft\elements\Category $element
     */
    protected function setAttributes($attributes, $element)
    {
```